src/EditorFeatures/Core.Wpf/QuickInfo/ProjectionBufferContent.cs

```diff
@@ -83,6 +82,9 @@ private IWpfTextView CreateView(ITextBuffer buffer)
             // Zoom out a bit to shrink the text.
             view.ZoomLevel *= 0.75;
 
+            // turn off highlight current line
```

Comment:
removing editable role didn't remove the current hightline feature. @olegtk told me, I had to remove some interactive role, but other roles we use, if I remove it, some taggers, or adornments or font and color stop working since they looks for those roles.

so, decide to explicitly disable it in the code.